### ISSUE TITLE
virtual-dom renderer?

### ISSUE THREAD

**Issue by paf31** [bodil/purescript-smolder#10]
I know this got removed but are there any plans to create a package with the old code?

**Comment by bodil** [comment]
Yes, I'm hoping to get around to it very soon, though if you've got plans for one yourself I won't hold you back. :)
